sources/packages/web/src/components/common/modals/ConfirmExtendTime.vue

```diff
@@ -0,0 +1,116 @@
+<template>
+  <ModalDialogBase
+    title="Confirm Extend Time"
+    dialogType="warning"
+    :showDialog="showDialog"
+    @click="extendTime"
+  >
+    <template v-slot:content>
+      <v-container class="p-component text-dark">
+        <p>
+          You are about to logged off, do you wish to extend your time?
+          <v-btn class="ml-2 text-white" color="warning" icon>
+            {{ countDown }}
+          </v-btn>
+        </p>
+      </v-container>
+    </template>
+    <template v-slot:footer>
+      <v-btn color="primary" outlined @click="dialogClosed"> No </v-btn>
+      <v-btn color="warning" depressed class="text-white" @click="extendTime">
+        <v-icon left size="25"> mdi-cancel </v-icon>
+        Yes
+      </v-btn>
+    </template>
+  </ModalDialogBase>
+</template>
+
+<script lang="ts">
+import ModalDialogBase from "@/components/generic/ModalDialogBase.vue";
+import { useModalDialog, useAuth, useClientLoader } from "@/composables";
+import { COUNT_DOWN_TIMER_FOR_LOGOUT } from "@/constants/system-constants";
+import { ref, onUnmounted, watch } from "vue";
+import { ClientIdType } from "@/types/contracts/ConfigContract";
+
+export default {
+  components: {
+    ModalDialogBase,
+  },
+  props: {
+    startTimer: {
+      type: Boolean,
+      required: false,
+    },
+    clientIdType: {
+      type: String,
+      required: true,
+      default: "" as ClientIdType,
+    },
+  },
+  setup(props: any) {
+    const countDown = ref(COUNT_DOWN_TIMER_FOR_LOGOUT);
```

Comment:
Not an issue at all, but countdown I actually one word. It could be all lower case 😉